.github/workflows/verifications.yml

```diff
@@ -47,6 +62,12 @@ jobs:
 
       - name: Install dependencies
         uses: bahmutov/npm-install@v1
```

Comment:
We have merged another PR removing  `npm-install` usage, can you adapt your changes to this?